### PR TITLE
add information about a-synchronic labeling to documentation

### PR DESCRIPTION
**Pull request by Rensvandeschoot**
This PR adds more detailed information to the documentation about the a-synchronic labeling/training.

### CODE REVIEW

**Comment by PeterLombaers** [comment]
Do you want to add this kind of information to a quick tour? To me it seems like the wrong place for it. A quick tour should not contain other information than how to use the basic functionalities.

File: docs/source/lab/oracle.rst [hunk 1]
```diff
@@ -93,6 +93,16 @@ You are asked to make a decision: relevant or irrelevant?
 While you review the documents, the software continuously improves its
 understanding of your decisions, constantly updating the underlying model.
 
+More specifically, each labeling decision of the user triggers a new model.
```
Comment (J535D165) [inline docs/source/lab/oracle.rst:96]:
```suggestion
More specifically, each labeling decision of the user starts the training of a new model given there is no model being trained at that time.
```

File: docs/source/lab/oracle.rst [hunk 2]
```diff
@@ -93,6 +93,16 @@ You are asked to make a decision: relevant or irrelevant?
...
+When this new model is done training, the unseen records' rank order is
```
Comment (J535D165) [inline docs/source/lab/oracle.rst:97]:
```suggestion
When this new model is trained, the unseen records' rank order is
```

File: docs/source/lab/oracle.rst [hunk 3]
```diff
@@ -93,6 +93,16 @@ You are asked to make a decision: relevant or irrelevant?
...
+updated. Training and labeling occur a-synchronic. With fast models, a new
+model will probably be ready before the user is done reading the text. With
+slower models, training continues until a new model is done, and the user can
+continue screening the next record in line (2nd, 3rd, etc.). Therefore, the
+record shown to the user can be the one with the highest relevance score of
+the new model or the highest-ranked as resulted from the old model until a new
+model is done.
```
Comment (J535D165) [inline docs/source/lab/oracle.rst:104]:
```suggestion
model is trained. 
```

File: docs/source/lab/oracle.rst [hunk 4]
```diff
@@ -93,6 +93,16 @@ You are asked to make a decision: relevant or irrelevant?
 While you review the documents, the software continuously improves its
 understanding of your decisions, constantly updating the underlying model.
 
+More specifically, each labeling decision of the user triggers a new model.
+When this new model is done training, the unseen records' rank order is
+updated. Training and labeling occur a-synchronic. With fast models, a new
+model will probably be ready before the user is done reading the text. With
```
Comment (J535D165) [inline docs/source/lab/oracle.rst:99]:
```suggestion
ranking will probably be available before the user finished reading the text. With
```

File: docs/source/lab/oracle.rst [hunk 5]
```diff
@@ -93,6 +93,16 @@ You are asked to make a decision: relevant or irrelevant?
...
+slower models, training continues until a new model is done, and the user can
```
Comment (J535D165) [inline docs/source/lab/oracle.rst:100]:
```suggestion
slower models, training continues until a new model is trained, and the user can
```

File: docs/source/lab/oracle.rst [hunk 6]
```diff
@@ -93,6 +93,16 @@ You are asked to make a decision: relevant or irrelevant?
...
+continue screening the next record in line (2nd, 3rd, etc.). Therefore, the
+record shown to the user can be the one with the highest relevance score of
+the new model or the highest-ranked as resulted from the old model until a new
```
Comment (J535D165) [inline docs/source/lab/oracle.rst:103]:
```suggestion
the second last model or the highest-ranked as resulted from the latest model until a new
```

The other way around?

File: docs/source/lab/oracle.rst (hunk 3 above)
Comment (J535D165) [inline docs/source/lab/oracle.rst:104]:
```suggestion
model is trained. 
```

**Review by J535D165 — COMMENTED** [review]
This is a good addition to the documentation. It explains the working of the complex frontend. I have some suggestions for the text.

**Pull request merged**